metricbeat/module/jolokia/_meta/Dockerfile

```diff
@@ -1,11 +1,39 @@
 # Tomcat is started to fetch Jolokia metrics from it
-FROM jolokia/java-jolokia:7
-ENV TOMCAT_VERSION 7.0.55
+FROM java:8-jdk-alpine
+
+ENV TOMCAT_VERSION 7.0.86
 ENV TC apache-tomcat-${TOMCAT_VERSION}
+ENV JOLOKIA_VERSION 1.5.0
+ENV PROXT_TARGET_TOMCAT_VERSION 9.0.7
+ENV PX_TC apache-tomcat-${PROXT_TARGET_TOMCAT_VERSION}
+
+RUN apk update;\
+    apk add curl
 
-HEALTHCHECK --interval=1s --retries=90 CMD curl -f curl localhost:8778/jolokia/
+HEALTHCHECK --interval=1s --retries=90 CMD curl -f localhost:8778/jolokia/
 EXPOSE 8778
-RUN wget -q http://archive.apache.org/dist/tomcat/tomcat-7/v${TOMCAT_VERSION}/bin/${TC}.tar.gz
-RUN tar xzf ${TC}.tar.gz -C /opt
 
-CMD env CATALINA_OPTS=$(jolokia_opts) /opt/${TC}/bin/catalina.sh run
+# Prepare a server where jolokia runs in proxy mode
+RUN wget http://archive.apache.org/dist/tomcat/tomcat-7/v${TOMCAT_VERSION}/bin/${TC}.tar.gz;\
+    tar xzf ${TC}.tar.gz -C /usr;\
+    sed -i -e 's/Connector port="8080"/Connector port="8778"/g' /usr/${TC}/conf/server.xml;\
+    wget http://central.maven.org/maven2/org/jolokia/jolokia-war/${JOLOKIA_VERSION}/jolokia-war-${JOLOKIA_VERSION}.war -O /usr/${TC}/webapps/jolokia.war
+
+# Set up a server to be connected in proxy mode
+RUN wget http://archive.apache.org/dist/tomcat/tomcat-9/v${PROXT_TARGET_TOMCAT_VERSION}/bin/${PX_TC}.tar.gz;\
+    tar xzf ${PX_TC}.tar.gz -C /usr;\
```

Comment:
Can you also remove these .tar.gz files that are not needed in the final image?

Reply:
I deleted the unnecessary tar.gz file from the final image.